pepper-apis/src/main/java/org/broadinstitute/ddp/json/FileUploadURLPayload.java

```diff
@@ -0,0 +1,63 @@
+package org.broadinstitute.ddp.json;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+import com.google.gson.annotations.SerializedName;
+
+public class FileUploadURLPayload {
+    @NotBlank
+    @SerializedName("fileName")
+    String fileName;
+
+    @NotBlank
+    @SerializedName("mimeType")
+    String mimeType;
+
+    @NotNull
```

Comment:
Consider using javax.validation.constraints.Positive and/or some other of the constraint annotations available.

Reply:
OK